Gododir/gen.go

```diff
@@ -563,26 +273,29 @@ func generateTasks(p *do.Project) {
 }
 
 func generate(service *service) {
-	codegen := "Go"
+	codegen := "--go"
 	if testGen {
-		codegen = "Go.TestGen"
+		codegen = "--go.testgen"
 		service.Fullname = strings.Join([]string{service.Fullname, testsSubDir}, string(os.PathSeparator))
 		service.Output = filepath.Join(service.Output, testsSubDir)
 	}
 
-	fmt.Printf("Generating %s...\n\n", service.Fullname)
-
-	delete(service)
+	fmt.Printf("Working on %s...\n\n", service.Fullname)
+	fmt.Printf("Generating on %s...\n\n", service.Fullname)
 
 	execCommand := "autorest"
 	commandArgs := []string{
-		"-Input", filepath.Join(swaggersDir, "azure-rest-api-specs", service.Input+"."+string(service.Extension)),
-		"-CodeGenerator", codegen,
-		"-Header", "MICROSOFT_APACHE",
-		"-Namespace", service.Name,
-		"-OutputDirectory", service.Output,
-		"-Modeler", string(service.Modeler),
-		"-PackageVersion", sdkVersion,
+		fmt.Sprintf("https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/%s", service.Input),
```

Comment:
So I take it we'll be generating from github instead of having to clone the specs repo.  Can we make this configurable with a default to github?  Also when we generate from github can we retrieve the hash so we can place it in the change log?